web/src/main/java/org/cbioportal/web/util/DataBinner.java

```diff
@@ -106,7 +96,7 @@ public DataBinner(DataBinHelper dataBinHelper,
         boolean numericalOnly = false;
 
         Range<BigDecimal> range = dataBinFilter.getStart() == null && dataBinFilter.getEnd() == null ?
```

Comment:
You added logic to dataBinHelper for when start and end are null. Here, you check that in this ternary, so you never pass null, null. Could you eliminate this check and change the behavior of `calcRange`, or are you doing `calcRange(null, null)` elsewhere?

Reply:
I was just moving `calcRange` from one class to another(for some spring dependency injection issues), wasn't checking what it was doing. Yeah, looks like this logic can be removed.

Reply:
Looks like `calcRange` is used even in other files which requires to check those fields for null